GPU/Common/TextureReplacer.cpp

```diff
@@ -337,6 +337,9 @@ bool TextureReplacer::LoadIniValues(IniFile &ini, VFSBackend *dir, bool isOverri
 		}
 	}
 
+	auto gr = GetI18NCategory(I18NCat::GRAPHICS);
+
+	g_OSD.Show(OSDType::MESSAGE_SUCCESS, gr->T("Texture replacement pack activated"));
```

Comment:
I forget if this can cause duplicates.  It should be at the end of LoadIni(), since LoadIniValues() can be called multiple times for region overrides.

-[Unknown]